src/control/server/README.md

```diff
@@ -163,205 +163,65 @@ In order to run the tool to perform administrative tasks, build and run the `dao
+The DAOS data plane utilises two forms of non-volatile storage, storage class memory (SCM) in the form of persistent memory modules and NVMe in the form of high-performance SSDs.
 
-### NVMe Controller and Namespace Discovery
+### SCM management capabilities
 
-The following animation illustrates starting the control server and using the management shell to view the NVMe Namespaces discovered on a locally available NVMe Controller (assuming the quickstart_guide instructions have already been performed):
+Operations on SCM persistent memory modules are performed using [go-ipmctl bindings](https://github.com/daos-stack/go-ipmctl) to issue commands through the ipmctl native C libraries.
 
-![Demo: List NVMe Controllers and Namespaces](/doc/graph/daosshellnamespaces.svg)
-
-### NVMe Controller Firmware Update
-
-The following animation illustrates starting the control server and using the management shell to update the firmware on a locally available NVMe Controller (assuming the quickstart_guide instructions have already been performed):
-
-![Demo: Updating NVMe Controller Firmware](/doc/graph/daosshellfwupdate.svg)
-
-### NVMe Controller Burn-in Validation
-
-Burn-in validation is performed using the [fio tool](https://github.com/axboe/fio) which executes workloads over the SPDK framework using the [fio_plugin](https://github.com/spdk/spdk/tree/v18.04.1/examples/nvme/fio_plugin).
-
-## SCM management capabilities
-
-### SCM Module Discovery
-
-### SCM Module Firmware Update
-
-### SCM Module Burn-in Validation
-
-Go app which can either be invoked with a subcommand to perform a specific task or without a subcommand which will launch an interactive shell.
-Command-line subcommands are implemented with the [go-flags](https://github.com/jessevdk/go-flags) package and interactive shell with [ishell](https://github.com/abiosoft/ishell).
-
-The management tool uses the [client API](../client) to interact with many [server](../server) instances as a gRPC client.
-The management tool has no storage library dependencies and as such is suitable to be run from a login node to interact with storage nodes.
-
-<details>
-<summary>Usage info from app help</summary>
-<p>
-
-```
-[tanabarr@ssh-1 ~]$ projects/daos_m/install/bin/daos_shell --help
-Usage:
-  daos_shell [OPTIONS] [command]
-
-Application Options:
-  -l, --hostlist=    comma separated list of addresses <ipv4addr/hostname:port> (default: localhost:10001)
-  -f, --hostfile=    path of hostfile specifying list of addresses <ipv4addr/hostname:port>, if specified takes preference over HostList
-  -o, --config-path= Client config file path
-
-Help Options:
-  -h, --help         Show this help message
+Provisioning SCM occurs by configuring modules in AppDirect memory regions (interleaved mode) in groups of modules local to a specific socket (NUMA) and resultant nvdimm namespaces are defined a device identifier (e.g. /dev/pmem0).
```

Comment:
As a DAOS newbie, it's not clear what the AppDirect memory regions are.  Perhaps add a link to a doc section that describes what it is for reference.   Does the sys-admin need to configure these modules, or does DAOS handle this configuration on their behalf?  Is this statement informational only, or is it intended to be instructional (i.e. the sys-admin should prep the SCM beforehand to get ready for DAOS in some way).  I know it's obvious to you, just not to me yet.  :-)